### PR TITLE
Fixing pages service version

### PR DESCRIPTION
**Pull request by votemike**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: app/config/services.yml [hunk 1]
```diff
@@ -5,6 +5,12 @@ services:
     cache.null_provider:
         class: Symfony\Component\Cache\Adapter\NullAdapter
 
+    cache.bbc:
+        class: BBC\ProgrammesPagesService\Cache\Cache
+        arguments:
+            - '@cache.programmes'
+            - ''
```
Comment (votemike) [inline app/config/services.yml]: What should this prefix be?

Comment (BPScott) [inline app/config/services.yml]: We'll want the app name, and release number

 `- 'clifton.%cosmos_component_release%'`

File: app/config/services.yml [hunk 2]
```diff
@@ -5,6 +5,12 @@ services:
     cache.null_provider:
         class: Symfony\Component\Cache\Adapter\NullAdapter
 
+    cache.bbc:
```
Comment (BPScott) [inline app/config/services.yml]: cache.* is the namespace that is created when you [configure the framework cache bits](http://symfony.com/doc/current/reference/configuration/framework.html#reference-cache). I don't think we should put this in there.

At the bottom of this file we define `pps.mapper_factory` and `pps.service_factory` as the utils that we need to setup the pages services. How about defining `pps.cache` down there too?

**Pull request merged**